pyro/distributions/transforms/neural_autoregressive.py

```diff
@@ -84,21 +102,29 @@ def log_abs_det_jacobian(self, x, y):
         return - 2. * (x - math.log(2.) + F.softplus(- 2. * x))
 
 
+def tanh():
```

Comment:
name overloaded?

Reply:
I think this is fine as long as it only lives in this file and `pyro.distributions.transforms`.